src/main/kotlin/no/nav/familie/ba/mottak/hendelser/JournalføringHendelseAivenConsumer.kt

```diff
@@ -0,0 +1,59 @@
+package no.nav.familie.ba.mottak.hendelser
+
+
+import io.micrometer.core.instrument.Counter
+import io.micrometer.core.instrument.Metrics
+
+import no.nav.joarkjournalfoeringhendelser.JournalfoeringHendelseRecord
+import org.apache.kafka.clients.consumer.ConsumerRecord
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.slf4j.MDC
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.kafka.annotation.KafkaListener
+import org.springframework.kafka.support.Acknowledgment
+import org.springframework.stereotype.Service
+import javax.transaction.Transactional
+
+@Service
+@ConditionalOnProperty(
+    value = ["funksjonsbrytere.kafka.enabled"],
+    havingValue = "true",
+    matchIfMissing = true
+)
+class JournalføringHendelseAivenConsumer(val journalhendelseService: JournalhendelseService) {
+
+
+    val journalføringshendelseAivenConsumerFeilCounter: Counter = Metrics.counter("joark.hendelse.journalføring.feil")
+
+    @KafkaListener(
+        id = "familie-ba-sak",
+        topics = ["\${JOURNALFOERINGHENDELSE_V1_TOPIC_AIVEN_URL}"],
+        containerFactory = "kafkaAivenHendelseListenerContainerFactory",
+        idIsGroup = false
+    )
+    @Transactional
+    fun listen(consumerRecord: ConsumerRecord<Long, JournalfoeringHendelseRecord>, ack: Acknowledgment) {
```

Comment:
I AivenConfig så er det konfigurert opp med 
ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG to StringDeserializer::class.java,

Hvis du skal bruke Avro, så må du nok conifigurere  opp egen kafkaconfig med AvroDesrializer som man gjør mot onprem.
value-deserializer: io.confluent.kafka.serializers.KafkaAvroDeserializer